bin/ci_to_gubernator.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Exit immediately for non zero status
+set -e
+# Check unset variables
+set -u
+
+SCRIPTPATH="$(cd "$(dirname "$0")" ; pwd -P)"
+ROOTDIR="$(dirname ${SCRIPTPATH})"
+
+if [ -z "$GCS_BUCKET_TOKEN" ]; then
+	echo "GCS_BUCKET_TOKEN unavailible"
+	exit 0
+fi
+
+TMP_SA_JSON=$(mktemp /tmp/XXXXX.json)
+ENCRYPTED_SA_JSON="${ROOTDIR}/.circleci/accounts/istio-circle-ci.gcp.serviceaccount"
+openssl aes-256-cbc -d -in "${ENCRYPTED_SA_JSON}" -out "${TMP_SA_JSON}" -k "${GCS_BUCKET_TOKEN}" -md sha256
+
+wget https://storage.googleapis.com/istio-tools/ci_to_gubernator/ci_to_gubernator
```

Comment:
I think you should be able to use go get here now that your tool has been submitted.

Reply:
I am still having issues with go get. 
```
$ go get istio.io/test-infra/toolbox/ci_to_gubernator
can't load package: package istio.io/test-infra/toolbox/ci_to_gubernator: found packages ci_to_gubernator (ci_to_gubernator.go) and main (main.go) in /Users/chx/go/src/istio.io/test-infra/toolbox/ci_to_gubernator
```
And that
```
$ go get istio.io/test-infra/toolbox/ci_to_gubernator/main
istio.io/test-infra/toolbox/ci_to_gubernator/main: cannot find package "istio.io/test-infra/toolbox/ci_to_gubernator/main" in any of:
	/usr/lib/google-golang/src/istio.io/test-infra/toolbox/ci_to_gubernator/main (from $GOROOT)
	/usr/local/google/home/chx/go/src/istio.io/test-infra/toolbox/ci_to_gubernator/main (from $GOPATH)
```
Perhaps this PR https://github.com/istio/test-infra/pull/777 helps resolve the issue. Please take a look.